clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -98,6 +98,13 @@ Improvements to clang-tidy
 New checks
 ^^^^^^^^^^
 
+- New :doc:`bugprone-incorrect-enable-shared-from-this
+  <clang-tidy/checks/bugprone/incorrect-enable-shared-from-this>` check.
+
+  Check if class/structs publicly inherits from ``std::enable_shared_from_this``,
```

Comment:
```suggestion
  Detects if class/structs publicly inherits from ``std::enable_shared_from_this``,
```
This is continuation of previous statement, so `check checks` does not sound well.

Reply:
Noted, will do.